src/frontend/containers/CreatePage.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC, useState } from 'react';
+// import { Path } from '../../shared/models/Path';
+import { Button, Form, Input, Label } from 'reactstrap';
+import { redirect } from 'react-router-dom';
+
+export const CreatePage: FC = () => {
+  const [path, setPath] = useState({
+    name: '',
+    description: '',
+  });
+  //when name updates
+  const nameChange = (name: string) => {
+    setPath({ ...path, name: name });
+  };
+
+  //when description updates
+  const descriptionChange = (description: string) => {
+    setPath({ ...path, description: description });
+  };
+
+  //creates new path and route to new page
+  const clickCreate = () => {
+    api
+      .createPath(path.name, path.description) //call to create path endpoint
+      .then((path) => {
+        //route to new page
+        redirect(`/view-path/${path.id}`);
+      })
+      .catch((error: unknown) => {
+        //catch error will be handled later
+        console.log(error);
+      });
```

Comment:
```suggestion
      .catch(captureError);
```

Doing this invokes the provided `captureError` function to present errors in a standard way.

Reply:
Do not actually accept this suggestion as you need to import the function from utils